conan/recipes/beman/all/test_package/src/beman/integration_test.cpp

```diff
@@ -0,0 +1,23 @@
+#include <example.hpp>
+#include <scn/scan.h>
+
+#include <iostream>
+
+int main()
+{
+    std::string source{
+        "1 2 3 4 5 6 7 8 9 11 22 33 44 55 66 77 88 99 111 222 333 444 555 666 "
+        "777 888 999"};
+    std::vector<int> integers;
+    auto input = scn::ranges::subrange{source};
+
+    while (auto result = scn::scan<int>(input, "{}")) {
+        integers.push_back(result->value());
+        input = result->range();
+    }
+
+    for (auto i : integers) {
+      std::cout << i << "\n";
+    }
```

Comment:
we probably can't do this bc it will break on so many compilers if we don't have c++23 but it sure is nicer to do ```std::print("{}", intergers);

Reply:
This is just a placeholder test for demonstration. Ideally we'd have a set of meaningful tests involving how different Beman Project libraries interact. Also, ideally, those tests would be at the top-level of this repo and be useable in various dependency management scenarios instead of being buried down in here in the Conan-specific logic.

Reply:
I've changed the implementation of the test package for the `beman` Conan package to use the tests located at the top of the `beman` git repo. This means the same tests are used across Conan and CMake+FetchContent workflows.